examples/arkane/networks/methoxy/input.py

```diff
@@ -2,11 +2,12 @@
 
 description = \
 """
-This example illustrates how to manually set up a CanTherm input file for a small P-dep reaction system [using only the
-RRHO assumption, and without tunneling, although this can be easily implemented]. Such a calculation is desireable if the user 
-wishes to supply experimentally determined freqeuncies, for example. Althgou some coommented notes below may be useful, 
-see http://greengroup.github.io/RMG-Py/users/cantherm/index.html for more documented information about CanTherm and 
-creating input files. (information pertaining this file is adopted by Dames and Golden, 2013, JPCA 117 (33) 7686-96.)
+This example illustrates how to manually set up an Arkane input file for a small P-dep reaction system (using only the
+RRHO assumption, and without tunneling, although this can be easily implemented). Such a calculation is desirable if
+the user wishes to supply experimentally determined frequencies, for example. Although some commented notes below may
+be useful, see http://greengroup.github.io/RMG-Py/users/arkane/index.html for more documented information about
```

Comment:
I think this URL should also be updated: http://reactionmechanismgenerator.github.io/RMG-Py/users/cantherm/index.html